optimum/commands/__init__.py

```diff
@@ -15,5 +15,4 @@
 from .base import BaseOptimumCLICommand, CommandInfo, RootOptimumCLICommand
 from .env import EnvironmentCommand
 from .export import ExportCommand, ONNXExportCommand, TFLiteExportCommand
-from .onnxruntime import ONNXRuntimeCommand, ONNXRuntimeOptimizeCommand, ONNXRuntimeQuantizeCommand
-from .optimum_cli import register_optimum_cli_subcommand
```

Comment:
Would need to make sure `register_optimum_cli_subcommand` is not used in subpackages

Reply:
Yes, and I was very much tempted to make it private to `optimum-cli`, actually.
There are no calls in `optimum` and I quickly checked the `optimum-XX` to verify that none of them was calling it either, but I was not 100 % sure.

Reply:
Existing subpackages are using the "register" method at the moment.